eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -130,3 +130,27 @@ jobs:
           testRunTitle: "$(OSName) - Browser - Integration Tests - [Node $(NodeVersion)]"
         condition: and(succeededOrFailed(), eq(variables['TestType'], 'browser'))
         displayName: "Publish browser integration test results"
+
+  - job: "TestSamples"
```

Comment:
[NIT] Maybe..
```suggestion
  - job: "Verify Samples"
```
(Take it if you like it)

Reply:
Unfortunately I think DevOps disallows using spaces in the job name :(

Reply:
Oh. Is it so? Spaces are allowed here though 🤔
![image](https://user-images.githubusercontent.com/10452642/68423336-c446a200-0156-11ea-9205-0d8453653c21.png)

Reply:
Yep, that's because it's a job with an associated matrix. The job name is "IntegrationTest" and the matrix entry name is "Linux_Node8", and it combines them with a space... No idea why this limitation is in place, it seems totally unnecessary.